AdaptivePELE/simulation/simulationrunner.py

```diff
@@ -36,6 +37,7 @@ def __init__(self):
         self.peleSteps = 0
         self.seed = 0
         self.exitCondition = None
+	self.time = None
```

Comment:
Indentation is off, and the same happens in other changes such as in validatorBlockNames and estimateDG